spec/services/participants/early_career_teacher_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+RSpec.describe Participants::EarlyCareerTeacher do
+  let(:klass) do
+    Class.new do
```

Comment:
Never seen it done like this, nice!

I usually just end up creating a class outside the `describe` block.